src/core/Vec.mli

```diff
@@ -80,4 +79,6 @@ val for_all : ('a -> bool) -> 'a t -> bool
 val pp :
   ?sep:string ->
   (Format.formatter -> 'a -> unit) ->
-  Format.formatter -> 'a t -> unit
+  Format.formatter ->
+  'a t ->
+  unit
```

Comment:
Again an example where the regularity of the code before made it clearer that this function takes a printing function, and returns one (though, in this case, this could be achieved by using a type alias for printing functions).